app/assets/javascripts/patients.coffee

```diff
@@ -37,6 +37,14 @@ ready = ->
   $(document).on "change", "#abortion-information-form input, #external_pledges input", ->
     updateBalance()
 
+  $(document).on "submit", "#generate-pledge-form form", ->
+    if($("#case_manager_name").val())
+      true
+    else
+      $("#generate-pledge-form .alert").show()
```

Comment:
Really, really love this -- very sharp of you to flag this and punch it into the workflow!